modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -48,6 +52,27 @@
 	public List<ServiceRegistration<?>> deploy(
 		ObjectDefinition objectDefinition) {
 
+		String restContextPath = objectDefinition.getRESTContextPath();
+
+		if (_companyLocalService.getCompaniesCount() > 1) {
```

Comment:
This is fine and forwarding for now, but the concept of default company exists you can check it... I think you can get it via the default group.

Reply:
Sorry. I was looking for this information in the Company services and database tables and I couldn't find it. I've added the last commit to avoid the namespace when the company is the default one